analytics_extension.go

```diff
@@ -0,0 +1,54 @@
+package helix
+
+// ExtensionAnalytic ...
+type ExtensionAnalytic struct {
+	ExtensionID string    `json:"extension_id"`
+	URL         string    `json:"URL"`
+	Type        string    `json:"type"`
+	DateRange   DateRange `json:"date_range"`
+}
+
+// DateRange ...
+type DateRange struct {
+	StartedAt Time `json:"started_at"`
+	EndedAt   Time `json:"ended_at"`
+}
+
+// ManyExtensionAnalytics ...
+type ManyExtensionAnalytics struct {
+	ExtensionAnalytics []ExtensionAnalytic `json:"data"`
+}
```

Comment:
We need to add the [`Pagination`](https://github.com/nicklaw5/helix/blob/master/helix.go#L97) field to this struct also, as the results can be paginated. For example:

```go
type ManyExtensionAnalytics struct {
	ExtensionAnalytics []ExtensionAnalytic `json:"data"`
	Pagination         Pagination          `json:"pagination"`
}
```